src/classes/UTIL_UnitTestData_TEST.cls

```diff
@@ -518,6 +551,20 @@ public class UTIL_UnitTestData_TEST {
         return attd;
     }
 
+    /*********************************************************************************************************
+    * @description Gets a Attendance Event with fields for testing.
+    * @param ContactId, CourseConnectionID, attendanceType, Reason 
```

Comment:
Can you split out the params so that each has its own definition, consistent with the other ApexDocs in this class?

Reply:
@robertbwatson Thanks for catching this, updated.